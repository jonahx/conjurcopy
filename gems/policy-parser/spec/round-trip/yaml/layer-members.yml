- !grant
  role: !automatic-role
    record: !layer dev/mysql
    role_name: use_host
  members:
    - !group developers
