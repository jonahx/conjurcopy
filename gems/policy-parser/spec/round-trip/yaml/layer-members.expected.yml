---
- !grant
  member:
  - !member
    role: !group
      id: developers
  role: !automatic-role
    record: !layer
      id: dev/mysql
    role_name: use_host
